java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -108,6 +109,26 @@ public void ping_with_message_returns_success() {
         assertEquals(message, pong);
     }
 
+    @SneakyThrows
+    @Test
+    public void unlink_returns_long_success() {
+        // setup
+        String[] keys = new String[] {"testKey1", "testKey2"};
```

Comment:
nit: Should we name the array args for consistency sake in all tests in this file?

Reply:
I prefer to name it after the name of the variable... makes it more "personal"?